target/classes/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
👀

Reply:
should I like, gitignore the whole target folder? not sure what it is tbh